### PR TITLE
#220: Update Drupal core to 9.0.x.

### PR DESCRIPTION
**Pull request by joeparsons**
## Description

- Updates Drupal core to version ~9.0.0~ 9.0.3
- ~Removes Drupal console because it is not compatible with Symfony 4 / Drupal 9~
- ~Pins CAS module to specific dev revision so composer doesn't complain about D9 compatibility~

## Related Issue
#220 

## How Has This Been Tested?
Locally with composer / lando.  The local packge/repo created by our lando config doesn't seem to be working correctly for me but a manual composer install using a local clone of the scaffolding repo worked for me.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart project in the right column.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by trackleft** [comment]
https://take.ms/29Z0N Nice https://7a16e9cd-6aff-413e-a49b-23f72085d7d6.probo.build/admin/reports/status

**Comment by tadean** [comment]
Tested locally and things look really promising! Amazing work @joeparsons !

File: package.json [hunk 1]
```diff
@@ -13,34 +13,37 @@
     "watch:js": "cross-env BABEL_ENV=quickstart node ./scripts/js/babel-es6-watch.js"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
```
Comment (tadean) [inline package.json:16]: In testing I ran into some issues with `yarn watch` not working. With the change here from `babel-core` to `@babel/core` it looks like the following change is necessary in line 3 of `scripts/js/compile.js`:
`const babel = require('@babel/core');`

**Comment by tadean** [comment]
(Re)tested locally and this seemed to work great @joeparsons ! Tested creation of content, drupal-check, unit tests, drush functions, js tooling, and some drupal console generation functions. The only issue I ran into is the `'@babel/core'` needed in `compile.js` listed above.

File: package.json (hunk 1 above)
Comment (joeparsons) [reply package.json:16]: Good catch.  Thanks @tadean!

**Pull request merged**